cmd/proxy/actions/home.go

```diff
@@ -5,5 +5,5 @@ import (
 )
 
 func proxyHomeHandler(c buffalo.Context) error {
-	return c.Render(200, proxy.HTML("index.html"))
+	return c.Render(200, proxy.JSON("Welcome to The Athen Proxy"))
```

Comment:
We should probably remove this handler as well, since you are asking for an API.

The `healthz` endpoint should be enough to verify that

Reply:
@manugupt1 We can keep this instead of returning a 404 on the home page which I know is ok for APIs usually.